docs/database/database-spans.md

```diff
@@ -268,6 +291,91 @@ Placeholders in a parameterized query SHOULD not be sanitized. E.g. `where id =
 e.g. from `IN (?, ?, ?, ?)` to `IN (?)`, as this can help with extremely long IN-clauses,
 and can help control cardinality for users who choose to (optionally) add `db.query.text` to their metric attributes.
 
+## Generating a summary of the query text
+
+The `db.query.summary` attribute captures a shortened representation of a query text
+which SHOULD have low-cardinality and SHOULD NOT contain any dynamic or sensitive data.
+
+> [!NOTE]
+> The `db.query.text` is intended to identify individual queries. Even though
+> is it sanitized if captured by default, it still could have high cardinality and
+> might reach hundreds of lines of code.
+>
+> The `db.query.summary` is intended to provide less granular grouping key that
+> can be used as a span name or a metric attribute in a common case. It SHOULD
+> only contain information that has a significant impact on the query, database,
+> or application performance.
+
+Instrumentations that support query parsing SHOULD generate query summary when
+one is not readily available from other sources.
+
+The summary SHOULD preserve the following parts of query in the order they were provided:
+- operations such as SQL SELECT, INSERT, UPDATE, DELETE or other statements
```

Comment:
is "statement" the right word here?

Reply:
yeah, not right. I changed to "command", but it's not a precise referenceable technical term either - I don't think there is a precise term defined in the language :( 

The problem is that we want a small subset of clauses - only the top-level ones. Open to any suggestions on how we can describe it better.
The only alternative I can think of is literally document all the commands that we'd support for SQL.

E.g. here's what ChatGPT  classifies as 'commands'

- SELECT
- INSERT
- UPDATE
- DELETE
- MERGE
- CREATE
- ALTER
- DROP
- TRUNCATE
- RENAME
- GRANT
- REVOKE
- COMMIT
- ROLLBACK
- SAVEPOINT
- SET TRANSACTION
- CREATE INDEX
- DROP INDEX
- ALTER INDEX
- START TRANSACTION
- SET

which seems to align with what we want it to be, so just saying "command" might be good enough.

Reply:
> "command" might be good enough

👍